docs/developers/NewToGluten.md

```diff
@@ -139,34 +140,37 @@ If download failed, delete this folder and try again.
 
 ### Set up project
 
-File->Open Folder   // select gluten folder
-Select cpp/CmakeList.txt as command prompt
-Select gcc version as command prompt
+- File->Open Folder   // select the Gluten folder
+- After the project loads, you will be prompted to "Select CMakeLists.txt". Select the `${workspaceFolder}/cpp/CmakeList.txt` file.
```

Comment:
${workspaceFolder}/cpp/CmakeList.txt` -> ${workspaceFolder}/cpp/CMakeLists.txt`. typo CMakeLists.txt

Reply:
Thank you for catching that, I pushed an update.